image_provisioner/inventory/hosts.AWS

```diff
@@ -32,6 +32,7 @@ config_repo=
 docker_login=no
 docker_storage_device=/dev/xvdb
 docker_storage_vgname=docker_vg
+docker_storage_driver=overlay
```

Comment:
Remember to add all new variables introduced in the roles to the inventory, even if they're just commented out. Specifically `update_kernel` should be added as well so we know that it exists.